### PR TITLE
Login

### PR DESCRIPTION
**Pull request by harsimran-preet**
Login page implemented

### CODE REVIEW

File: backend/server.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
 const backend = require("./backend");
+const express = require("express");
+//import { errorHandler, notFound } from "./middleware/errorMiddleware.js";
 const PORT = process.env.PORT || 5000;
+const app = express(); // main thing
+
+// app.use(notFound);
+// app.use(errorHandler);
```
Comment (ujwal-jain) [inline backend/server.js:8]: Remove unneeded comments

File: frontend/src/App.js [hunk 2]
```diff
@@ -20,18 +21,25 @@ function App() {
         let result = await axios.get("http://localhost:5000/user", userCred);
         setUser(result["data"]["user"]);
       } catch (error) {
-        console.log(error);
+        // console.log(error);
```
Comment (ujwal-jain) [inline frontend/src/App.js:24]: Remove unneeded comments

File: frontend/src/components/login.component.js [hunk 3]
```diff
@@ -1,49 +1,108 @@
...
+        username: username,
+        password: password,
+      },
+    };
+    let result = await getUser(userCred);
+    if (result.status === 200) {
+      setAuthenticated(true);
+    }
+    // if (result.statusCode === 200)
+    // console.log(username, password);
```
Comment (ujwal-jain) [inline frontend/src/components/login.component.js]: Remove unneeded comments

**Review by ujwal-jain — COMMENTED** [review]
LGTM, just remove the unneeded comments

**Comment by JihoPark21** [comment]
As Ujwal said remove the test comments but besides that it looks good to me

**Comment by ujwal-jain** [comment]
Also need to resolve merge conflicts

**Pull request merged**